src/Root.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { HashRouter as Router, Route, Routes } from 'react-router-dom';
+import { App } from './App';
+import { ErrorPage } from './pages/ErrorPage/ErrorPage';
+import { HomePage } from './pages/HomePage/HomePage';
+import { PeoplePage } from './pages/PeoplePage/PeoplePage';
+
+export const Root: React.FC = () => (
+  <Router>
+    <Routes>
```

Comment:
![image](https://github.com/mate-academy/react_people-table-basics/assets/22824431/6d2fbece-e5ef-4755-a350-87b86b089a13)

Reply:
`replace` attribute not applied yet (to `Navigate`). Try using a back button in your browser after navigating to `/home`